src/components/Shows/index.js

```diff
@@ -41,15 +117,22 @@ export class Shows extends React.Component {
       return <Loader />;
     }
 
-    return <React.Fragment>{showPreviewList}</React.Fragment>;
+    return (
+      <div>
+        <div className={styles.container}>{this.createCheckboxes()}</div>
+
+        <React.Fragment>{showPreviewList}</React.Fragment>
```

Comment:
`React.Fragment`only needs to be used when you don't want to wrap a component in an outer `div`. Check it out here: https://reactjs.org/docs/fragments.html. I see that I used it incorrectly in the previous version of the file aswell 😅. It can be safgely removed here 👍